notebooks/MAST/TESS/beginner_tess_exomast/beginner_tess_exomast.ipynb

```diff
@@ -167,7 +167,9 @@
     "\n",
     "First, we provide some background.  TESS does a search of the postage-stamp, two-minute cadence data for transit signals. For every signal it identifies with its Transit Planet Search (TPS) module, it creates something called a Threshold Crossing Event (TCE). TCEs are periodic signals that exceed a nominal signal-to-noise requirement. Some are consistent with transiting planets, others are eclipsing binaries, and others are more consistent with variable stars or noise in the data. The exported products include a data validation time series file that includes a median detrended time series as well as the time series it used to do the planet search.\n",
     "\n",
-    "Here we retrieve the header and data associated with the TCE that corresponds to the ephemeris of WASP-18 b from the data validation time series files. The info request provides the header information from the primary header and the data header for the specified TCE.  Note: this can be done for any TCE, not just those associated with known planets."
+    "Here we retrieve the header and data associated with the TCE that corresponds to the ephemeris of WASP-18 b from the data validation time series files. The info request provides the header information from the primary header and the data header for the specified TCE.  Note: this can be done for any TCE, not just those associated with known planets.\n",
+    "\n",
+    "We must first retrieve the TCE information to determine which TESS sectors contain WASP-18 b, and pick a particular sector range to query."
```

Comment:
Can we add a bit more explanation here:
"Because many TESS targets are observed in more than one sector and the TESS mission does multi-sector searches, we must first determine which sector's search found a TCE on WASP-18. We can then choose from those sector ranges for our query.

Reply:
Done.